Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -2196,6 +2198,8 @@ class RenderStyle {
         unsigned autosizeStatus : 5;
 #endif
         // 51 bits
+        unsigned hasSetFillColor : 1;
```

Comment:
Shouldn't this be in the "rare inherited" data structure ? (with text fill and stroke color)

Reply:
The name is misleading, this is not about SVG fill-color, it's about the CSS color property. It's not that uncommon for websites to set the text color afaik, so it shouldn't be in rareData

Reply:
Yes indeed, color is stored `m_inheritedData`.

Reply:
Should we change the name then ?

Reply:
Yes, though I did express disagreement with this general approach on Slack.